raiden/tests/unit/test_transfer.py

```diff
@@ -158,16 +157,17 @@ def test_transfer(raiden_network):
     assert isinstance(a1_recv_messages[0], DirectTransfer)
 
 
+@pytest.mark.timeout(10)
```

Comment:
If the transfer logic is broken and the transfer does not complete then `result.wait()` will wait forever and the test never fails nor succeeds.  The timeout forces the test to fail.